addon/helpers/list.js

```diff
@@ -1,6 +1,16 @@
 import { MicroState } from 'ember-microstates';
+import assign from '../utils/assign';
 
 export default MicroState.extend({
+
+  wrap(value) {
+    return assign(value, {
+      valueOf() {
+        return value.slice();
```

Comment:
Not quite sure what the reason for this is, other than perhaps so that it won't have the actions on it?